docs/fds-components/index.html

```diff
@@ -1 +1,5 @@
-<!DOCTYPE html><html><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1.0"><meta http-equiv="X-UA-Compatible" content="ie=edge"><title>fds-components</title></head><body><div id="rsg-root"></div><script src="build/bundle.5380be7d.js"></script></body></html>
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
Merge mistake

Reply:
😨 Fortunately, everything in `docs/` is generated by the build. I can just rebuild & commit. Nice catch!